### DISCUSSION TITLE
2021-06-23

### DISCUSSION THREAD

**Discussion by haudiobe** [Dash-Industry-Forum/webRTC#3] [Minutes]
# Agenda Call 2021-06-23
## Roll Call
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/tsto/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/tsto/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

Name | Attendee Email | Join Time | Attendance Duration
-- | -- | -- | --
Thomas Stockhammer | tsto@qti.qualcomm.com | 2021-06-23 06:59:46 | 62 mins
Phillip Maness Xperi | phillip.maness@xperi.com | 2021-06-23 06:59:29 | 62 mins
Yuriy Reznik | yreznik@brghtcove.com | 2021-06-23 06:59:29 | 62 mins
Ofer Shem Tov | ost@verizonmedia.com | 2021-06-23 07:00:11 | 61 mins
Richard Doherty | richard.doherty@dolby.com | 2021-06-23 07:00:27 | 61 mins
Iraj Sodagar | Tencent | 2021-06-23 07:01:36 | 60 mins
Piers O'Hanlon | piers.ohanlon@bbc.co.uk | 2021-06-23 07:01:27 | 60 mins
Bill Wishon | bill.wishon@phenixrts.com | 2021-06-23 07:02:41 | 59 mins
Ali | q@q.ca | 2021-06-23 07:03:20 | 58 mins
Andy Rosen - SequenceKey.com | arosen@sequencekey.com | 2021-06-23 07:03:46 | 58 mins
Charles Qualcomm | clo@qti.qualcomm.com | 2021-06-23 07:03:02 | 58 mins
Emre Aksu | emre.aksu@nokia.com | 2021-06-23 07:02:52 | 58 mins
Mike Dolan | md.1@newtbt.com | 2021-06-23 07:03:59 | 58 mins
Johan THEO Technologies | johan.vounckx@theoplayer.com | 2021-06-23 07:05:06 | 55 mins
Julia Kenyon Phenix | julia.kenyon@phenixrts.com | 2021-06-23 07:06:44 | 55 mins
Michael Stattmann | michael.stattmann@castlabs.com | 2021-06-23 07:05:51 | 55 mins
Kashyap Kammachi Sreedhar | kashyap.kammachi-sreedhar@nokia.com | 2021-06-23 07:07:56 | 54 mins
Mickael Raulet | m.raulet@ateme.com | 2021-06-23 07:07:58 | 54 mins
Pankaj | pc.chaudhari@hulu.com | 2021-06-23 07:10:27 | 51 mins
asolo | alexander.solonsky@casdtlabs.com | 2021-06-23 07:17:13 | 44 mins
Brian Lee Dolby | brian.lee@dolby.com | 2021-06-23 07:21:27 | 40 mins
Nicol So | nicol.so@commscope.com | 2021-06-23 07:21:51 | 39 mins
Nicol So | nicol.so@commscope.com | 2021-06-23 07:21:51 | 39 mins
Ted Laverty | ted.laverty@xperi.com | 2021-06-23 07:22:55 | 37 mins
Michael Demb Telestream | michael.demb@telestream.net | 2021-06-23 07:02:11 | 33 mins
Patrick Gendron Harmonic | patrick.gendron@harmonicinc.com | 2021-06-23 07:02:01 | 32 mins
Giri Mandyam | mandyam@qti.qualcomm.com | 2021-06-23 07:33:56 | 26 mins
Bram Tullemans | tullemans@ebu.ch | 2021-06-23 07:50:37 | 11 mins
Julia Kenyon | julia.kenyon@phenixrts.com | 2021-06-23 07:00:56 | 6 mins
Kashyap Kammachi Sreedhar | kashyap.kammachi-sreedhar@nokia.com | 2021-06-23 07:02:02 | 6 mins

</body>

</html>

## Minutes from last meeting: https://github.com/Dash-Industry-Forum/webRTC/discussions/2
  - please check still
## [Policy](https://github.com/Dash-Industry-Forum/webRTC/blob/main/README.md#dash-if-policy)
  - agreeable. We run under those policy.
## Review [logistics](https://github.com/Dash-Industry-Forum/webRTC/blob/main/README.md#logistics)
  - Ali explains the logistics
  - no questions, no comments
## Review [scope](https://github.com/Dash-Industry-Forum/webRTC/blob/main/README.md#scope)
  - Ali explains the scope
  - Yuriy: Session handling refers to what? 
    - Session establishment
    - Session handling
    - Session negotiation and setup is likely the most relevant work. You can use http or webRTC.
  - Yuriy: Are we using WebRTC with full A/V stack or just data channel?
    - it is not data channel from Phenix perspective, full AV stack is used.
    - not excluded per se, but this is not in the primary scope
  - Iraj: please add reference to the slides in the scope.
  - Michael: Is it about browser stack oder more webRTC delivery protocol?
    - specifically we are trying to solve is session negotiation such clients can setup different platforms.
    - separate problem of discovery the streaming experience, re-use MPD as the description
    - this would be on top of webRTC, more on a data model level.
    - We should make clear that this is on using webRTC, not to change it.
  - Andy: is this about dash.js extension?
    - this is not the really the scope initially
  - Piers: Is conversion in scope?
    - not really, more about switching across streams.
    - conversion use cases can be added to Google
  - Ali: SDP can also be used for multi-angle video source and such use cases. Why not use SDP? IETF MMUSIC working group is for this purpose
    - Prevalence of MPD knowledge in the premium media ecosystem is the primary objective instead of the using the MPD
    - This sounds simpler than changing it to SDP
    - SDP allows to negotiate codecs and resolutions that are supported.
  - WebRTC may be more suitable for managed networks
  - Instead of DRM, would call it annotation features. This is part of what we need.
  - Multicast may also be an important use case - W3C started some work on this.
## Start drafting report (Google docs)
    - Julia, Ali and Thomas setup a document with the existing text. 
    - https://docs.google.com/document/d/1vwtJCgE95d2bPthwfKlvGxPiElgqJQZf7rtz_g_ZgZw/edit?usp=sharing 
## Action Items
   - we will update and refine the scope based on the discussion
   - Creating an outline of the report with adding information 
   - Ali mails to IETF MOPs and SVA
   - Please use github for discussion and setting up issues.

## AOB

**Comment by acbegen** [comment]
The link for the W3C document (Piers): https://www.w3.org/TR/webrtc-nv-use-cases/#toc, indeed if something relevant comes out of this exploration, we should let W3C know.

**Reply by pmaness** [reply]
This github page is PUBLIC, and I get a TON of spam from nefarious sources. It's really a bad idea to list everyone's email like this. The spam bots just roll through and collect them. Please modify the attendance list to not include email addresses.
